Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -248,6 +248,15 @@ final class LayerTests: XCTestCase {
         XCTAssertEqual(output.shape, expected)
     }
 
+
+    func testFunction() {
```

Comment:
Could you please make this be the last test in this file? We expect the ordering of tests in `allTests` to reflect exactly the ordering of test case function declarations.

Reply:
@rxwei  I thought we were grouping the tests in a file together, Since i added this to the Core.swift file, i thought it would belong to that class of tests, am I wrong?